lib/twilio/rtc/rtcpc.ts

```diff
@@ -88,8 +97,8 @@ RTCPC.prototype.createAnswer = function(maxAverageBitrate, codecPreferences, con
     const sdp = setMaxAverageBitrate(answer.sdp, maxAverageBitrate);
 
     return promisifySet(this.pc.setLocalDescription, this.pc)(new RTCSessionDescription({
-      type: 'answer',
       sdp: setCodecPreferences(sdp, codecPreferences),
+      type: 'answer',
```

Comment:
Seems that this PR has a lot of line order changes? Why not just keep the lines in original order if the code isn't modified?

Reply:
Good observation @kpchoy . We want the linter to pass to have green build at a minimum. This includes line order changes that you observed - we sort keys and params alphabetically.